README.md

```diff
@@ -9,20 +9,23 @@ A process wrapper script that monitors the execution of a command.
 ```shell
 > duct --help
 
-usage: duct [-h] [-p OUTPUT_PREFIX] [--sample-interval SAMPLE_INTERVAL]
+<!--- BEGIN HELP -->
+usage: duct [-h] [--version] [-p OUTPUT_PREFIX]
+            [--sample-interval SAMPLE_INTERVAL]
             [--report-interval REPORT_INTERVAL] [-c {all,none,stdout,stderr}]
             [-o {all,none,stdout,stderr}]
             [-t {all,system-summary,processes-samples}]
-            command [arguments ...]
+            command ...
 
 Gathers metrics on a command and all its child processes.
 
 positional arguments:
   command               The command to execute.
-  arguments             Arguments for the command. (default: None)
+  inner_args            Arguments for the command.
```

Comment:
in above `usage:` output there is now no `inner_args`, only `...` . Ideally should be matched here

Reply:
Updated in last push, hows this?

`duct --help`

```
usage: duct [-h] [--version] [-p OUTPUT_PREFIX] [--sample-interval SAMPLE_INTERVAL] [--report-interval REPORT_INTERVAL]
            [-c {all,none,stdout,stderr}] [-o {all,none,stdout,stderr}] [-t {all,system-summary,processes-samples}]
            command [inner_args ...] ...

Gathers metrics on a command and all its child processes.

positional arguments:
  command [inner_args ...]
                        The command to execute, along with its inner arguments.
  inner_args            Arguments for the command.
```

Reply:
sorry for being pain, but `inner_args` is like a jargon which IMHO hard to communicate. Could you consistently rename it to `command_args`?

Reply:
No pain, command_args is better, +1 Fixed in latest push